bitnami/rabbitmq-cluster-operator/values.yaml

```
@@ -661,6 +664,9 @@ msgTopologyOperator:
     ##   - myRegistryKeySecretName
     ##
     pullSecrets: []
+  ## @param msgTopologyOperator.revisionHistoryLimit sets number of replicaset to keep in k8s
+  ##
+  revisionHistoryLimit: 3
```

What if we honor the Kubernetes default value which is `10`?
```suggestion
  revisionHistoryLimit: 10
```